awx/main/credential_plugins/tss.py

```diff
@@ -1,7 +1,10 @@
 from .plugin import CredentialPlugin
 from django.utils.translation import gettext_lazy as _
 
-from thycotic.secrets.server import DomainPasswordGrantAuthorizer, PasswordGrantAuthorizer, SecretServer, ServerSecret
+try:
+    from delinea.secrets.server import DomainPasswordGrantAuthorizer, PasswordGrantAuthorizer, SecretServer, ServerSecret
+except ImportError:
+    from thycotic.secrets.server import DomainPasswordGrantAuthorizer, PasswordGrantAuthorizer, SecretServer, ServerSecret
```

Comment:
Hi @delinea-sagar 

I know that this was already merged, but can you please explain why did you add this `ImprotError` check?

There is a PR which tries to follow the same pattern, but I wonder if it is actually required.

Related PR -> #14362

Reply:
@andrii-zakurenyi this is a common pattern that we follow